charts/datahub/subcharts/acryl-datahub-actions/README.md

```diff
@@ -39,4 +40,4 @@ Current chart version is `0.0.3`
 | datahub.metadata_service_authentication.enabled | bool | `false` | Whether Metadata Service Authentication is enabled. |
 | global.datahub.metadata_service_authentication.systemClientId | string | `"__datahub_system"` | The internal system id that is used to communicate with DataHub GMS. Required if metadata_service_authentication is 'true'. |
 | global.datahub.metadata_service_authentication.systemClientSecret.secretRef | string | `nil` | The reference to a secret containing the internal system secret that is used to communicate with DataHub GMS. Required if metadata_service_authentication is 'true'. |
-| global.datahub.metadata_service_authentication.systemClientSecret.secretKey | string | `nil` | The key of a secret containing the internal system secret that is used to communicate with DataHub GMS. Required if metadata_service_authentication is 'true'. |
```

Comment:
What's the diff here? Was it intentional?

Reply:
A newline was added here. The diff doesn't show this properly somehow...
The other READMEs also have an empty newline at the end of the file.